config/routes.rb

```diff
@@ -1,10 +1,7 @@
-require 'staff_constraint'
-
 require 'sidekiq/web'
 
 Rails.application.routes.draw do
-  mount Peek::Railtie => '/peek',    constraints: StaffConstraint.new unless Rails.env.test?
-  mount Sidekiq::Web  => '/sidekiq', constraints: StaffConstraint.new
+  mount Peek::Railtie => '/peek'
```

Comment:
@mastahyeti I have it set to staff only here: https://github.com/education/classroom/blob/master/app/controllers/application_controller.rb#L18

But I can put it under the `stafftools` namespace which does verify that the user is properly authenticated.